scripts/install.sh

```diff
@@ -52,6 +52,9 @@ function install {
 		ARCH="amd64"
 	elif uname -m | grep arm > /dev/null; then
 		ARCH="arm" #TODO armv6/v7
+		if OS == "Darwin"; then
+			OUT_DIR='/bin'
```

Comment:
isn't `/bin` for system binaries?

i think dangerous to use `/bin` because you might accidentally overwrite your `ls` program

Reply:
This is due to the new mac upgrades moving everything to /bin. Its the new "apple way to do things" which is why i made sure we were on darwin before changing the out_dir first. linux machines are as you described.